packages/discovery-provider/plugins/pedalboard/apps/backfill-audio-analyses/src/redis.ts

```diff
@@ -0,0 +1,98 @@
+import { createClient, RedisClientType } from 'redis'
+import { config } from '.'
+import { logger } from './logger'
+
+let redisClient: RedisClientType
+let isReady: boolean
+
+const parseArray = (json: string | null) => {
```

Comment:
No need to change but I wonder if we could pull some of these utilities from a shared location? Or is this all unique to this plugin

Reply:
yeah i copied this from another plugin; i only see it in the solana-relay rn so prob fine to just leave it